src/common-components/tests/BaseComponent.test.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import { IntlProvider } from '@edx/frontend-platform/i18n';
+
+import SmallScreenLayout from '../SmallScreenLayout';
+import MediumScreenLayout from '../MediumScreenLayout';
+import LargeScreenLayout from '../LargeScreenLayout';
+
+describe('ScreenLayout', () => {
+  it('should display the form, pass as a child in SmallScreenLayout', () => {
+    const smallScreen = mount(
+      <IntlProvider locale="en">
+        <SmallScreenLayout>
+          <form>
+            <input type="text" />
+          </form>
+        </SmallScreenLayout>
+      </IntlProvider>,
+    );
+    expect(smallScreen.find('form').exists());
```

Comment:
This statement itself does not verify anything, if `exists()` returned false this test will not fail. You need to verify it against true like this:
```suggestion
    expect(smallScreen.find('form').exists()).toEqual(True);
```